custodia/secrets.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015  Custodia Project Contributors - see LICENSE file
+# (C) 2015  Custodia Project Contributors - see LICENSE file
```

Comment:
if you're gonna change the Copyright notice, might as well change the date too :P